libr/bin/p/bin_vsf.c

```diff
@@ -75,6 +77,12 @@ static bool load(RBinFile *bf, RBuffer *b, ut64 loadaddr) {
 		}
 		// read all VSF modules
 		offset = sizeof (struct vsf_hdr);
+		ut8 vice_version[sizeof (VICE_VERSION)];
+		if (r_buf_read_at (bf->buf, offset, vice_version, sizeof (VICE_VERSION)) == sizeof (VICE_VERSION)) {
+			if (!memcmp (vice_version, VICE_VERSION, sizeof (VICE_VERSION) - 1)) {
```

Comment:
not sure about this -1?

Reply:
that's because the string constant `VICE_VERSION` is nul-terminated, while the string in the binary is not, same as at the `CMP_MODULE` macro a few lines below

Reply:
Ok! Good to confirm